### ISSUE TITLE
2.8.8 drop JSON support? ORA-40569: Unimplemented JSON feature.

### ISSUE THREAD

**Issue by viplifes** [sijms/go-ora#540]
Version 2.8.7 work fine!

Versions 2.8.8 - 2.8.11 have error: ORA-40569: Unimplemented JSON feature.

```golang
package main

import (
	"database/sql"
	"fmt"

	go_ora "github.com/sijms/go-ora/v2"
)

func main() {

	connOra, err := sql.Open("oracle", go_ora.BuildUrl("127.0.0.1", 1521, "ORCLPDB1", "SYS", "dbpaSS1#", nil))
	if err != nil {
		panic(err)
	}
	defer connOra.Close()
	run(connOra)
	// print
	// OCIBlobLocator OCIBlobLocator
}

func run(conn *sql.DB) {
	conn.Exec(`GRANT CREATE SESSION TO SYS`)
	conn.Exec(`DROP TABLE IF EXISTS types_test`)
	conn.Exec(`CREATE TABLE types_test (myblob BLOB NULL, myjson JSON NULL)`)
	conn.Exec(`INSERT INTO types_test (myblob, myjson)VALUES(UTL_RAW.CAST_TO_RAW('blobval - not json'), json('{"foo": "bar"}'))`)

	rows, err := conn.Query(`SELECT * FROM types_test`)
	if err != nil {
		panic(err)
	}
	cTypes, _ := rows.ColumnTypes()
	fmt.Println(cTypes[0].DatabaseTypeName(), cTypes[1].DatabaseTypeName())
}

```

**Comment by sijms** [comment]
HI @viplifes 
I run the code with multiple version of go-ora (latest, v2.8.7, 2.7.21) and all failed at this stmt:
```sql
INSERT INTO types_test (myblob, myjson)VALUES(UTL_RAW.CAST_TO_RAW('blobval - not json'), json('{"foo": "bar"}'))
```
with this error
```bash
ORA-40445: unsupported JSON operation
```
to confirm I run same statement on SQL Developer and get same error
my database version is 19c

**Comment by sijms** [comment]
when I change the insert statement and remove json keywords everything is ok
```sql
INSERT INTO types_test (myblob, myjson)VALUES(UTL_RAW.CAST_TO_RAW('blobval - not json'), '{"foo": "bar"}'); 
```

**Comment by viplifes** [comment]
I use docker image hub.docker.com/gvenzl/oracle-free:23-slim

https://hub.docker.com/r/gvenzl/oracle-free

**Comment by sijms** [comment]
fixed in next release

**Comment by sijms** [comment]
fixed v2.8.13
